src/types/index.ts

```diff
@@ -0,0 +1,438 @@
+import { OpenmrsResource } from "@openmrs/esm-framework";
+
+export enum SearchTypes {
+  BASIC = "basic",
+  ADVANCED = "advanced",
+  SEARCH_RESULTS = "search_results",
+  SCHEDULED_VISITS = "scheduled-visits",
+  VISIT_FORM = "visit_form",
+  QUEUE_SERVICE_FORM = "queue_service_form",
+  QUEUE_ROOM_FORM = "queue_room_form",
+}
+
+export interface Patient {
+  uuid: string;
+  display: string;
+  identifiers: Array<any>;
+  person: Person;
+}
+
+export interface Attribute {
+  attributeType: OpenmrsResource;
+  display: string;
+  uuid: string;
+  value: string | number;
+}
+
+export interface Person {
+  age: number;
+  attributes: Array<Attribute>;
+  birthDate: string;
+  gender: string;
+  display: string;
+  preferredAddress: OpenmrsResource;
+  uuid: string;
+}
+export interface AppointmentsFetchResponse {
+  data: Array<Appointment>;
+}
+
+export interface Appointment {
```

Comment:
These might have come from the queue esm and should be cleaned to avoid leaving in unecessary code